.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           path: target
           key: clippy-target-${{ runner.os }}-${{ steps.rust-version.outputs.version }}-${{ hashFiles('Cargo.lock') }}
       - name: Run clippy
-        run: cargo clippy --all --all-targets
+        run: cargo clippy --all --all-targets -- -A clippy::upper_case_acronyms
```

Comment:
Why did this change? If it's because of some difference between the FIPS and non-FIPS version, please use an attribute on the FFI module instead, rather than blanket allowing this for the whole workspace.